### PR TITLE
Remove old SlidePane examples and add new ones 

### LINKED ISSUE #1019: Example: Slide Pane
Add a set of examples for the Slide Pane and remove the old examples from the repository.

### PR DESCRIPTION
**Pull request by JamesLMilner**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/framework/blob/master/STYLE.md)
* [ ] Unit tests are included in the PR
* [ ] For new widgets, an entry has been added to the `.dojorc`
* [ ] Any widget variant uses `theme.compose` like [this](https://github.com/dojo/widgets/issues/847)
* [ ] WidgetProperties are exported

**Description:**

Adds new examples for SlidePane and removes old ones

Resolves #1019

### CODE REVIEW

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-nf5cpnwvw.now.sh
* Tests Deployment: https://dojowidgets-akmh3qrmt.now.sh

File: src/examples/src/widgets/slide-pane/BottomAlignSlidePane.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import SlidePane, { Align } from '@dojo/widgets/slide-pane';
+import icache from '@dojo/framework/core/middleware/icache';
+
+const factory = create({ icache });
+
+export default factory(function BottomWidthSlidePane({ middleware: { icache } }) {
+	return (
+		<div>
```
Comment (tomdye) [inline src/examples/src/widgets/slide-pane/BottomAlignSlidePane.tsx]: is this div needed?

**Comment by tomdye** [comment]
seems there's some extra divs in these examples, could you remove them please?

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-qkno88g27.now.sh
* Tests Deployment: https://dojowidgets-h66e4upwf.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-qkno88g27.now.sh
* Tests Deployment: https://dojowidgets-h66e4upwf.now.sh

**Pull request merged**